extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -341,6 +352,11 @@ export class AmpLiveList extends AMP.BaseElement {
     if (shouldSendAmpDomUpdateEvent) {
       promise = promise.then(() => {
         this.sendAmpDomUpdateEvent_();
+        const event = createCustomEvent(
+          this.win,
+          'amp-live-list.update',
+          {insertedItemIds: insertItemIds, replacedItemIds: replaceItemIds});
```

Comment:
@sebastianbenz How would you feel about not including the IDs? I'm having a hard time coming up with a good use case for this as well.